### ISSUE TITLE
Explicitly set the default app server user to 'nobody'

### ISSUE THREAD

**Issue by brent-hartwig** [project-lux/lux-marklogic#252]
> [!NOTE]
> One with ML admin console is able to verify this change.

Recently discovered the default user for both the request-group-1 and request-group-2 app servers were set to an admin user, as opposed to 'nobody', who does not have the ability to authenticate.  Scope of this ticket is to explicitly set the default user within the ML Gradle configuration file for both app servers, which get used via the `mlDeployServers` deployment task or higher.

**Comment by brent-hartwig** [comment]
Implemented in PR https://github.com/project-lux/lux-marklogic/pull/254 and merged into the release1.22 branch.

**Comment by roamye** [comment]
Approved by UAT 8/7

cc: @prowns

**Comment by brent-hartwig** [comment]
Closing.  I confirmed Green-as-PROD has blank values for the default user in both app servers.  While 'nobody' was expected, blank is the equivalent.  I ran `mlDeployServers` in DEV today, after setting the default user to myself in both app servers; after the task completed, both had a value of 'nobody'.  I'm not enough concerned to investigate further.  cc: @kamerynB, @xinjianguo, @clarkepeterf, @roamye
